ENIGMAsystem/SHELL/Graphics_Systems/General/GSbackground.h

```diff
@@ -17,9 +17,10 @@
 #ifndef ENIGMA_GSBACKGROUND_H
 #define ENIGMA_GSBACKGROUND_H
 
-#include "Universal_System/scalar.h"
 #include "GScolors.h"
 
+#include "Universal_System/scalar.h"
```

Comment:
I did this because `GScolors.h` is more likely to include `scalar.h` than the other way around... hence the former should be included first (you know, in case your compiler is too stupid to figure that out...).